akka-docs/src/main/paradox/guide/actors-intro.md

```diff
@@ -72,7 +72,7 @@ This is a very simple model and it solves the issues enumerated previously:
  * There is no need for locks. Modifying the internal state of an actor is only possible via messages, which are
    processed one at a time eliminating races when trying to keep invariants.
  * There are no locks used anywhere, and senders are not blocked. Millions of actors can be efficiently scheduled on a
-   dozen of threads reaching the full potential of modern CPUs. Task delegation is the natural mode of operation for actors.
+   dozen threads reaching the full potential of modern CPUs. Task delegation is the natural mode of operation for actors.
```

Comment:
I agree with @raboof that "a small number of threads" is more what we mean.

(12 threads on a 32 core box will not be enough, so we don't want to be very precise)